### DISCUSSION TITLE
Starting MonaiLabel server with Multiple inputs

### DISCUSSION THREAD

**Discussion by Ivan-E-Johnson** [Project-MONAI/MONAILabel#1472] [General]
I have been working with 4D Multimodal registered data in MonaiBundle and am able to define a Datalist with multiple paths as the input for one "Image". Is there a way to set up the MonaiLabel application to accept a Datalist like Monaibundle does? If not would a secondary script that just appends the datastore.json file that is created when I pass it a path to the standard studies directory?
Any suggestions would be greatly appreciated! Thank you in advance.

For clarity Here is the file structure I am working with:
Studies Directory:
├── sub1
│ ├── Co-registered images
│ │ ├── modality_1.nii.gz
│ │ ├── modality_2.nii.gz
│ │ └── modality_3.nii.gz
│ └── label images
└── sub2

In Monai bundle I can pass it a list of dictionaries organized like so:

{
"image": [
"path_to_modality_1",
"path_to_modality_2",
"path_to_modality_3"
],
"label": "path_to_label"
}

**Comment by diazandr3s** [comment]
Hi @Ivan-E-Johnson,

This is a great initiative. I saw you started a solution for this here: https://github.com/Project-MONAI/MONAILabel/pull/1474
A nice use case for this is when using the BRATS dataset. Are you thinking about the same?

**Reply by SachidanandAlle** [reply]
Yes . You can..

The most simple way is.. you zip all 4 images and have your image loader to load the zip file accordingly..

**Reply by diazandr3s** [reply]
Agree. The "issue" comes when you want to visualize all 4 modalities in Slicer. I thought it was a solution for this :)
